audb/core/api.py

```diff
@@ -316,6 +353,12 @@ def exists(
     Returns:
         ``True`` if database flavor exists
 
+    Example:
+        >>> audb.exists('emodb', version='1.1.1')
+        True
```

Comment:
Maybe I should write the load command here again to make it more obvious why the flavor exists in the cache?

Reply:
I updated it accordingly as I think it makes it much easier to understand and does not add a speed burden as the data is already in cache.